addons/sale_project/models/project.py

```diff
@@ -403,6 +403,16 @@ def _get_revenues_items_from_sol(self, domain=None, with_action=True):
             'total': {'to_invoice': total_to_invoice, 'invoiced': total_invoiced},
         }
 
+    def _get_revenues_items_from_invoices_domain(self, domain=None):
+        if domain is None:
+            domain = []
+        return expression.AND([
+            domain,
+            [('move_id.move_type', 'in', ['out_invoice', 'out_refund']),
```

Comment:
(Thanks to one of your last fixes) we could use `get_sale_types` method. 🙂 
```suggestion
            [('move_id.move_type', 'in', self.env['account.move'].get_sale_types()),
```

What do you think? 🙂